tests/ce/test_ce.py

```diff
@@ -0,0 +1,33 @@
+from openfisca_us_data import CE
+
+
+def test_ce_from_2019():
+    """Test the Consumer Expenditure data generating functions for 2019"""
+    ce = CE()
```

Comment:
Yeah here we also don't need the instantiation step - everything should work the same from the actual class. It works with the instance of the class too, because all the same attributes are properties.

Reply:
initiation step gone.